c10/core/DispatchKey.h

```diff
@@ -77,6 +77,9 @@ enum class DispatchKey : uint8_t {
   VE, // For out of tree & closed source integration of SX-Aurora / NEC
   Lazy, // For lazy tensor backends
 
+
+  Tracer,
```

Comment:
Not sure what you're trying to do here as there is not description in the PR.
But moving this is going to break a lot of things! (as you can see in the CI).

Reply:
@albanD hey sorry I didn't notice that you got auto added to this diff that's just a POC. If you're curious the point is to surface stuff like `aten::resize_` in the ir (ctrl+f `resize_` in [this paste](https://www.internalfb.com/phabricator/paste/view/P459794273)).